### ISSUE TITLE
LP_PWD is displaying \[.     \] always.  No path information shown.

### ISSUE THREAD

**Issue by woo37830** [nojhan/liquidprompt#579]
Shell: bash
Operating system: OS X 15.1
Liquid Prompt version (tag, commit): eda83ef - (HEAD -> master, origin/master, origin/HEAD)

pwd always give path, but the prompt using the default set does not show anything about the path in the prompt.  echo $LP_PWD displays as in the title.

**Comment by Rycieos** [comment]
What is your bash version? The version of bash that ships with modern macOS (if you are using that) is very old.

I'm assuming you mean your OS is macOS 10.15.1. This version is still in beta, and one of the changes is a switch from bash to zsh as the default shell. This might have something to do with it.

If you actually are running zsh, make sure to include the version.

**Comment by woo37830** [comment]
echo $0 gives -bash
bash --version gives:
GNU bash, version 3.2.57(1)-release (x86_64-apple-darwin19)
Copyright (C) 2007 Free Software Foundation, Inc.

**Comment by woo37830** [comment]
I also did a >exec /bin/zsh and tried.  I had to re-source my liquidprompt/liquidprompt and got the same results, i.e. the path information is not filled in.  My git status shows and other information.

**Comment by woo37830** [comment]
AHA!  I have found the problem!  In Terminal it works fine and I'm using bash.  In iTerm2, which I have been using instead of Terminal for awhile, it does NOT work.  The startup for iTerm2 seems to follow the same logic of loading the script files as Terminal, but somehow does not handle the paths in the prompt.  I will go back to using Terminal until that gets cleared up.

**Comment by woo37830** [comment]
I posted notes to the Issues for liquid prompt.  The problem appears to be that I was using iTerm2 instead of Terminal.  In Terminal, it works fine. 

My >echo $0 returned -bash
and bash —version returned 
GNU bash, version 3.2.57(1)-release (x86_64-apple-darwin16)
Copyright (C) 2007 Free Software Foundation, Inc.
I tried while using iTerm2 to use zsh, but it made no difference.  Been working with bash too long to convert.
